kod/object/item.kod

```diff
@@ -613,6 +615,61 @@ messages:
       return 1;
    }
 
+   GetRarity()
+   {
+      local count;
+      count = 0;
```

Comment:
It doesn't look like this value is ever used.

Reply:
Removed `count = 0` since it was pointed out it is never used.